src/WinReg.jl

```diff
@@ -1,130 +1,32 @@
+#-----------------------------------------------------------
+#-----------------------------------------------------------
+# Define function to access windows registry
+#
+#File: jWinReg.jl
+#Author: Leonardo Cocco
+#Creation Date: 11-04-2016
+#Last update 21-11-2016
+#ver: 0.12
+#----------------------------------------------------------
```

Comment:
Correct. It's my habit to use this header before I use versioning system, and I've forgot to delete them